src/main/java/org/concordion/internal/command/executeCommand/ExecuteCommand.java

```diff
@@ -35,52 +126,43 @@ public void removeExampleListener(ExampleListener listener) {
 
 
     @Override
-    public List<CommandCall> getExamples(CommandCall command) {
-        List<CommandCall> superExamples =  super.getExamples(command);
-
-        ExecuteCommandStrategy executeCommandStrategy = getStrategy(command);
-        List<CommandCall> strategyExamples = executeCommandStrategy.getExamples(command);
-
-        List<CommandCall> allExamples = new ArrayList<CommandCall>();
-        allExamples.addAll(superExamples);
-        allExamples.addAll(strategyExamples);
-
-        return allExamples;
-    }
-
-    @Override
     public void execute(CommandCall commandCall, Evaluator evaluator, ResultRecorder resultRecorder) {
         ExecuteCommandStrategy executeCommandStrategy = getStrategy(commandCall);
-        executeCommandStrategy.execute(commandCall, evaluator, resultRecorder);
+        if (executeCommandStrategy != null) {
+            executeCommandStrategy.execute(commandCall, evaluator, resultRecorder);
+        } else {
+            commandCall.getChildren().execute(evaluator, resultRecorder);
+        }
     }
 
     private ExecuteCommandStrategy getStrategy(CommandCall commandCall) {
         ExecuteCommandStrategy executeCommandStrategy;
-        if (commandCall.getElement().isNamed("table")) {
-            executeCommandStrategy = new ExecuteCommandTableStrategy(this);
-        } else if (commandCall.getElement().isNamed("ol") || commandCall.getElement().isNamed("ul")) {
-        	executeCommandStrategy = new ExecuteCommandListStrategy();
+        if (isTableElement(commandCall)) {
+            return null;
+        } else if (isListElement(commandCall)) {
+            executeCommandStrategy = new ExecuteCommandListStrategy();
         } else {
             executeCommandStrategy = new ExecuteCommandDefaultStrategy(this);
         }
         return executeCommandStrategy;
     }
 
+
     void announceExecuteCompleted(Element element) {
-    	for (ExecuteListener listener : executeListeners) {
-    		listener.executeCompleted(new ExecuteEvent(element));
-		}
+        for (ExecuteListener listener : executeListeners) {
+            listener.executeCompleted(new ExecuteEvent(element));
+        }
     }
 
     protected void announceBeforeExample(String exampleName, Element element, ResultRecorder resultRecorder) {
```

Comment:
Actually, because we're doing the example command properly now, the announce before and example methods will be called by the ExampleCommand. So I think we can remove those code from the ExecuteCommand class.